pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ['setuptools', 'wheel', 'pytest', 'numpy', 'ase', 'hsd']
```

Comment:
`pytest`, `numpy`, `ase` and `hsd` are not setup requirements but runtime requirements and shouldn't be listed here.